Presentation.Web/Tests/01_Parallel/It-Contract/LocalAdmin.ItContract.UiCustomization.e2e.spec.ts

```diff
@@ -0,0 +1,78 @@
+﻿import Login = require("../../Helpers/LoginHelper");
+import TestFixtureWrapper = require("../../Utility/TestFixtureWrapper");
+import ContractHelper = require("../../Helpers/ContractHelper");
+import NavigationHelper = require("../../Utility/NavigationHelper");
+import ContractNavigationSrefs = require("../../Helpers/SideNavigation/ContractNavigationSrefs");
+
+describe("Local admin is able customize the IT-Contract UI", () => {
+
+    var loginHelper = new Login();
+    var testFixture = new TestFixtureWrapper();
+    var navigation = new NavigationHelper();
+
+    beforeAll(() => {
+        testFixture.enableLongRunningTest();
+    });
+
+    afterAll(() => {
+        testFixture.cleanupState();
+        testFixture.disableLongRunningTest();
+    });
+
+    var localContractPath: string | null = null;
+
+    it("Disabling Tabs/fields will hide the tabs/fields on the IT-Contract details page", () => {
+        var contractName = createName("contractName");
+
+        return loginHelper.loginAsGlobalAdmin()
```

Comment:
This test is missing all of the custom org stuff from the system version of the test. without the org scoping it will randomly fail since it will conflict with other tests running on the same org


Notice that the global admin is created as local admin in the created org 
![image](https://user-images.githubusercontent.com/13116889/172543237-5a6bc379-05df-4364-9ff1-911d2dbf0784.png)

and then during local system activation::::>

![image](https://user-images.githubusercontent.com/13116889/172543279-fa6110c5-5ed5-4bc8-8956-16d9efb47764.png)

Notice how it switches org.

**So what you need** is to create the org, add local admin to the global admin and then you must create a contract while in the org.. and THEN you can execute the tests.